Makefile

```diff
@@ -50,12 +50,13 @@ UPSTREAM_TAG ?= latest
 # BUILD_DEPLOY_IMAGE_TAG is the docker tag from uselagoon/build-deploy-image to use -
 # latest is the most current release
 # edge is the most current merged change
+BUILD_DEPLOY_IMAGE_REPO = uselagoon/build-deploy-image
 BUILD_DEPLOY_IMAGE_TAG ?= edge
 
 # UI_IMAGE_REPO and UI_IMAGE_TAG are an easy way to override the UI image used
 # only works for installations where INSTALL_STABLE_CORE=false
-# UI_IMAGE_REPO = uselagoon/ui
-# UI_IMAGE_TAG =
+UI_IMAGE_REPO = uselagoon/ui
+UI_IMAGE_TAG = main
```

Comment:
acknowledging that this override needs to stay here until the next stable release is out of the door to ensure Keycloak library backwards compatibility